ContactDetailsApi.Tests/V1/Controllers/ContactDetailsControllerTests.cs

```diff
@@ -70,5 +77,37 @@ public void GetContactDetailsByTargetIdThrowsException()
 
             func.Should().Throw<ApplicationException>().WithMessage(exception.Message);
         }
+
```

Comment:
Maybe we should add a not found test here

Reply:
I'll add one here yes, good call.

Reply:
I am not sure what not found would mean on the basis of a contact actually?